packages/terra-date-input/tests/jest/DateInput.test.jsx

```diff
@@ -2,14 +2,21 @@ import React from 'react';
 /* eslint-disable-next-line import/no-extraneous-dependencies */
 import { shallowWithIntl, renderWithIntl, mountWithIntl } from 'terra-enzyme-intl';
 import ThemeContextProvider from 'terra-theme-context/lib/ThemeContextProvider';
-
+import { v4 as uuidv4 } from 'uuid';
 import DateInput from '../../src/DateInput';
 import DateInputField from '../../src/DateInputField';
 import DateInputUtil from '../../lib/DateInputUtil';
 
-jest.mock('uuid', () => ({ v4: () => '00000000-0000-0000-0000-000000000000' }));
-
 describe('DateInput', () => {
+  let mockSpyUuid;
+  beforeAll(() => {
+    mockSpyUuid = jest.spyOn(uuidv4, 'v4').mockImplementation(() => '00000000-0000-0000-0000-000000000000');
+  });
+
+  afterAll(() => {
+    mockSpyUuid.mockRestore();
+  });
```

Comment:
Instead of doing this twice, I believe we can do it once outside the describe block.

Reply:
It works, fixed here: https://github.com/cerner/terra-framework/pull/1765/commits/6711ca76a70529f38ca38ff4fce256a990abb5bb

This commit also replaces `mockImplementation` with `mockReturnValue`